tests/omise/res/obj/OmiseObjectTest.php

```diff
@@ -6,6 +6,19 @@ class OmiseObjectTest extends TestConfig
 {
     public static $obj;
 
+    /**
+     * @test
+     */
+    public function build_a_new_object_using_given_data()
+    {
+        $object = OmiseCharge::build('chrg_test', array('object' => 'charge', 'id' => 'chrg_test'));
```

Comment:
If we're building a charge, passing `'object'=>'charge'` in the values seems kinda crazy - this could be done automatically? i.e. put an object type class constant on the OmiseCharge class etc.